.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
               PATHS=sql
             else
               echo "Check dry run for modified queries"
-              PATHS="$(git diff origin/master... --name-only -- sql)"
+              PATHS="$(git diff origin/master... --name-only --diff-filter=d -- sql)"
```

Comment:
Needed to add this `diff-filter` because the CI logic here was detecting the deleted file, and trying to dry run it, though it no longer exists. `--diff-filter=d` means exclude deleted files.